File: validator-app/src/main/java/org/cna/keyple/famoco/validator/di/AppModule.java
package org.cna.keyple.famoco.validator.di;

import android.content.Context;

import org.cna.keyple.famoco.validator.Application;

import dagger.Binds;
import dagger.Module;

@Module
public abstract class AppModule {
    // expose Application as an injectable context
    @Binds
    abstract Context bindContext(Application application);
}

File: validator-app/src/main/kotlin/org/cna/keyple/famoco/validator/di/AppModule.kt
package org.cna.keyple.famoco.validator.di

import android.content.Context
import dagger.Binds
import dagger.Module
import org.cna.keyple.famoco.validator.Application

@Module
abstract class AppModule {
    // expose Application as an injectable context
    @Binds
    abstract fun bindContext(application: Application): Context
}